crates/sncast/tests/helpers/fixtures.rs

```diff
@@ -90,7 +90,7 @@ pub async fn deploy_latest_oz_account() {
     .await;
 }
 pub async fn deploy_ready_account() {
-    let provider = get_provider(URL).expect("Failed to get the provider");
+    let provider = get_provider(&Url::parse(URL).unwrap()).expect("Failed to get the provider");
```

Comment:
Cannot we parse `URL` somewhere globally so it can just be used in tests? It's quite annoying to have to always do `Url::parse(URL).unwrap()`.

Reply:
Okay, if we also use it as a string, we can keep it.